README.md

```diff
@@ -66,6 +66,9 @@ a look at it for sure!
 Feel free to open any issue you find too (but a PR is always better :P )
 
 ## Release Notes/Contributors/Etc
+Release 0.2.0
```

Comment:
Not sure if it's documented or not but does this match the root project policy?

Reply:
I didn't find information on how to increment the version number, so I took a stab at it.